app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,141 @@
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) =>
+    roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase()),
+  )
+
+  //  get the progress bar colours for the current box in the progress bar
+  const progressBarColourObject = Object.values(
+    getProgressBarColors(currentBox, 'purple.20'),
+  )
+
+  return (
+    <>
+      <HStack mt={-10} mb={5} alignItems={'flex-end'}>
+        <Heading size="md" fontFamily="title" adjustsFontSizeToFit>
+          My Experience
+        </Heading>
+        <Flex flex={1} mt={-5} alignItems={'flex-end'}>
+          <StaThemeLogo />
+        </Flex>
+      </HStack>
+      <ProgressBar colours={progressBarColourObject} />
+      <View margin={'4'}>
+        <FreeSearchBar
+          marginTop="2"
+          marginBottom="-2"
+          handleSubmit={() => null}
+          handleChangeText={(text: string) => setSearchTxt(text)}
+        />
+      </View>
+      <ScrollView>
+        <VStack mt={-2} margin="5">
+          {filteredSkills.map((roleGroup: RoleGroup, index: number) => (
+            <View
+              key={`${roleGroup.groupName} - ${index}`}
+              flexDirection={'row'}
+              alignItems={'center'}
+            >
+              <Checkbox
+                onChange={() => {
+                  // required to make the checkbox work on android devices (ios should work without this)  https://reactnative.dev/blog/2022/12/13/pointer-events-in-react-native
+                  if (skillsValue.includes(roleGroup.groupName)) {
+                    setSkillsValue(
+                      skillsValue.filter(
+                        skill => skill !== roleGroup.groupName, // remove skill from the array
+                      ),
+                    )
+                  } else {
+                    setSkillsValue([...skillsValue, roleGroup.groupName]) // add skill to the array
+                  }
+                }}
+                checked={skillsValue.includes(roleGroup.groupName)}
+              />
+              <Text
+                ml="4"
+                marginY="0.5"
+                onPress={() => {
+                  // required to make the text label work on android devices (ios should work without this)
+                  if (skillsValue.includes(roleGroup.groupName)) {
+                    setSkillsValue(
+                      skillsValue.filter(
+                        skill => skill !== roleGroup.groupName,
+                      ),
+                    )
+                  } else {
+                    setSkillsValue([...skillsValue, roleGroup.groupName])
+                  }
+                }}
+              >
+                {roleGroup.groupName}
+              </Text>
+            </View>
+          ))}
+        </VStack>
+        <VStack marginX={5}>
+          <ProfileButtons
```

Comment:
It's my bad for not including it in the list of custom components [here](https://github.com/Scottish-Tech-Army/Volunteer-app/blob/develop/APP_DEVELOPMENT.md#custom-components) (sorry about that), but there's already a `SecondaryButton` component which I think might be styled to do what you need here with the outline?  (so would save you having to set all these settings manually on the button -- and more reusable)

Reply:
Also, from looking at Mia's designs on Figma, is the idea that the button has the outline until at least one checkbox is ticked, and then it becomes solid purple?  (i.e. the outlined version is a disabled state look)  Is getting the button to change that way within the scope of this ticket?

Reply:
Thanks @joefhall. Wasn't aware of the SecondaryButton. Will look into this change.